source/access/managing_disk_usage.rst

```diff
@@ -1,22 +1,43 @@
 .. _disk usage:
 
+###############################
 How much disk space am I using?
-===============================
+###############################
 
 .. _quota:
 
 Total disk space used on file systems with quota
-------------------------------------------------
+================================================
 
 On file systems with quota enabled, you can check the amount of disk space that
 is available for you, and the amount of disk space that is in use by
 you.
 
-On most systems, ``myquota`` will show you for the ``$VSC_HOME``,
-``$VSC_DATA`` and ``$VSC_SCRATCH`` file systems either the
-percentage of the available disk space you are using, or the
-absolute amount. Users from Ghent university should check their disk usage
-using the `web application <https://account.vscentrum.be/>`_.
+VSC account page
+----------------
+
+Your VSC account page shows up-to-date information about your storage usage on
+each of the file systems accessible to your account. This information is
+available in the *Usage* section of the
+`View Account <https://account.vscentrum.be>`_ tab and includes:
+
+* total data size
```

Comment:
I'm not sure we (at KU Leuven) have set this up, for my personal vsc account the Used column is n/a.

Reply:
Can you verify if that is intended or just a bug on your side?

Reply:
It seems we don't send any data about this. We will see internally if we can set this up, for now you can ignore this comment.